### PR TITLE
fix: use `release-plz-` rather than `release-plz/` branch prefix

### LINKED ISSUE #1017: Don't use `/` in branch name.
## Motivations

https://camp.smolnet.org/@frost/111166858853943185

- Would you like to implement this feature? [y/n]

## Solution

Use - instead. 

Make sure the change is retrocompatible.

### PR DESCRIPTION
**Pull request by eopb**
fixes #1017

### CODE REVIEW

File: crates/release_plz_core/src/pr.rs [hunk 1]
```diff
@@ -2,7 +2,8 @@ use chrono::SecondsFormat;
 
 use crate::PackagesUpdate;
 
-pub const BRANCH_PREFIX: &str = "release-plz/";
+pub const BRANCH_PREFIX: &str = "release-plz-";
```
Comment (MarcoIeni) [inline crates/release_plz_core/src/pr.rs:5]: I renamed this from `release-plz_` to `release-plz-`, so that the branch contains only `-` separators.

**Comment by MarcoIeni** [comment]
Thanks Ethan! I did a small change so that old release PRs continue to work as expected and new PRs have the new branch name :)

**Comment by MarcoIeni** [comment]
Ethan, what have you done, you broke release-plz! 😛

**Comment by eopb** [comment]
Thanks @MarcoIeni Sorry I made a mess :laughing:. I wasn't able to run the tests locally so didn't notice

**Comment by MarcoIeni** [comment]
> Sorry I made a mess

Ahah I was just joking :)

Also since you are a new contributor, the CI didn't trigger 👍 
I was confused because I saw a green checkmark on your commit, so I thought it was a flaky test 😅

**Pull request merged**